doc/source/templates/05_dreambooth_finetuning/configs/compute/gce.yaml

```diff
@@ -0,0 +1,7 @@
+# 4 A100 GPUs with 40gb GRAM each
+# This is a bit overkill, but instances with L4/A10G GPUs are not yet available on GCE
+head_node_type:
+  name: head_node_type
+  instance_type: a2-highgpu-2g-nvidia-a100-40gb-4
```

Comment:
Do we know when these other instance types be available?
There's some more work to be done to get this example running on a single GPU (using deepspeed / gradient checkpointing / some other tricks). Let's do that in a follow-up.
 
https://cloud.google.com/compute/docs/gpus#l4-gpus